<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="text">
        <string>Zurück</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonForward">
       <property name="text">
        <string>Vorwärts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReload">
       <property name="text">
        <string>Neu Laden</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editAddress">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Hier die URL eingeben</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
